### ISSUE TITLE
When using  bridge runtime directly, it breaks easyadminbundle :  Unable to add global "ea" as the runtime or the extensions have already been initialized 

### ISSUE THREAD

**Issue by allan-simon** [brefphp/symfony-bridge#58]
With this error , https://github.com/EasyCorp/EasyAdminBundle/issues/3715#issuecomment-999616605 

(the work around seems to work) 

I think it's because the process is kept alive so some stuff are kept initialized and not cleaned across request ?

**Comment by mnapoli** [comment]
> I think it's because the process is kept alive so some stuff are kept initialized and not cleaned across request ?

Yes, that means that EasyAdmin is incompatible with keeping the process alive. So your options seem to be:

- don't use EasyAdmin
- find a way to fix EasyAdmin
- don't keep the PHP process alive (don't use `BREF_LOOP_MAX`)

**Comment by allan-simon** [comment]
ok make sense,  to the extent of your knowledge, there's no way to have a safety on our side/bref side , to force clean before and/or after a request ? (like  unset global variables etc. )  ?

**Comment by t-richard** [comment]
The safety on Bref side is not using BREF_LOOP_MAX 😅

There's no way to know what needs cleanup and what doesn't, that is up to the library to decide. That's for the same reason that Symfony provides some tools to make it easier for you to handle such cases but can't handle it in a generic way that would make all code compatible with things like roadrunner, swoole and the likes.

**Comment by allan-simon** [comment]
@mnapoli  it seems to come with a significant performance improvement and stack simplification , so why not ? 

why did you try to create bref though php is not by default compatible with AWS Lambda ? :)

**Comment by allan-simon** [comment]
@mnapoli  I'm revisiting this issue but now I know the "recipe" to solve it 

 1. short term:  in my codebase decorate services that are problematic and make them implement the kernel.reset tag
 2. long term: try to backport that directly in said libraries


I'm just missing one piece  how do I test easily locally this use case using the bref cli and/or docker containers ?

i.e just changing the env variable for the `bref/php-81-fpm-dev:2`  containers to

```
        'HANDLER' => 'App\\Kernel',
        'DOCUMENT_ROOT' => 'public',
        'BREF_LOOP_MAX' => '1000',
        'BREF_BINARY_RESPONSES' => '1',
```

does not seem enough

**Comment by allan-simon** [comment]
ah thanks I missed the link to https://github.com/brefphp/dev-server
